bin/util.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import print_function
```

Comment:
Very happy with that! But then the scripts should have python3 in the shebang right?